classification.sh

```diff
@@ -80,13 +87,14 @@ if [ ! -f $inputFile ]; then
     exit 1
 fi 
 
-mkdir -p $SCRIPTDIR/temp_features;
+mkdir $tmpDir;
```

Comment:
Isn't the `-p` argument necessary during the first script execution, when `$SCRIPTDIR` (the parent folder) is missing?

Reply:
The `-p` argument was re-integrated as after testing, the elimination was not necessary.